keyboards/coseyfannitutti/mullet/readme.md

```diff
@@ -0,0 +1,15 @@
+# mullet
+
+![mullet](imgur.com image replace me!)
+
+A short description of the keyboard/project
```

Comment:
```suggestion
A 68-key keyboard with USB Type-C and RGB underglow.
```

Reply:
Could you please replace the description?